scripts/release/release.sh

```diff
@@ -139,90 +144,43 @@ prepare_package() {
 		../elektra-$VERSION
 	make
 	make run_all
-	make run_memcheck
+	# memleak because of external tools on debian buster
+	# discussed in https://github.com/ElektraInitiative/libelektra/pull/3530
+	make run_memcheck || true
 
-	# Now copy the packages to ftp:
-	cp $BUILD_DIR/elektra-$VERSION.tar.gz* $FTP_DIR/releases
-	cp $FTP_DIR/releases/elektra-$VERSION.tar.gz $BASE_DIR/$VERSION/
+	cp $BUILD_DIR/elektra-$VERSION.tar.gz* $BASE_DIR/$VERSION/
 }
 
-configure_debian_package() {
+configure_package() {
 	echo "Configuring debian package..."
```

Comment:
```suggestion
	echo "Configuring package..."
```

Or "building package"? (requires renaming of function, too)